core/sqf/sql/scripts/install_local_hadoop

```diff
@@ -1554,7 +1554,7 @@ echo "$MY_LOCAL_SW_DIST/${HBASE_TAR}"
       org.apache.hadoop.hbase.security.visibility.VisibilityController"
   fi
 
-  if [[ "$HBASE_DISTRO" != "APACHE1.2" || "$HBASE_DISTRO" != "CDH5.7"]]; then 
+  if [[ "$HBASE_DISTRO" != "APACHE1.2" && "$HBASE_DISTRO" != "CDH5.7"]]; then 
```

Comment:
There needs to be a space added after "CDH5.7". Without that some shells give a syntax error.